### PR TITLE
[release/3.1] Add missing Oracle RIDs

### PR DESCRIPTION
**Pull request by Mr-Tao**
This change adds previously missing RIDs for OL releases 7, 8 and 9 and respective RHEL counterparts.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/corefx?pullRequest=43138) <br/>All CLA requirements met.

**Comment by carlossanlop** [comment]
@ViktorHofer @ericstj is the comment about signing the license agreement a blocker for approving changes?

**Comment by ericstj** [comment]
Yes, we require signing the CLA to accept contributions.  Also this should first be approved in main before backporting to release/3.1.

File: pkg/Microsoft.NETCore.Platforms/runtimeGroups.props [hunk 1]
```diff
@@ -124,17 +129,17 @@
     <RuntimeGroup Include="rhel">
       <Parent>linux</Parent>
       <Architectures>x64</Architectures>
-      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6</Versions>
+      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6;7.7;7.8;7.9;7.10</Versions>
```
Comment (omajid) [inline pkg/Microsoft.NETCore.Platforms/runtimeGroups.props:132]: These minor versions were needed back in the early releases of .NET Core, which happened to treat each minor version of RHEL as a separate distinct version. With recent improvements (all of which were implemented by 3.1), that's not needed anymore. .NET will convert version like `7.X` to the major version - `7` - internally and save us extra maintenance workload. I don't think there's any reason to add these to the RID graph now.

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.9_release_notes/overview says:

>  Red Hat Enterprise Linux 7.9 is the last minor release of RHEL 7. 

So we shouldn't be adding 7.10 anyway.

File: pkg/Microsoft.NETCore.Platforms/runtimeGroups.props [hunk 2]
```diff
@@ -124,17 +129,17 @@
...
       <Parent>linux</Parent>
       <Architectures>x64;arm64</Architectures>
-      <Versions>8;8.0;8.1</Versions>
+      <Versions>8;8.0;8.1;8.2;8.3;8.4;8.5;8.6</Versions>
     </RuntimeGroup>
     <RuntimeGroup Include="rhel">
       <Parent>linux</Parent>
       <Architectures>x64;arm64</Architectures>
-      <Versions>9</Versions>
+      <Versions>9;9.0;9.1</Versions>
```
Comment (omajid) [inline pkg/Microsoft.NETCore.Platforms/runtimeGroups.props:142]: :-1:  on adding RHEL 9 minor versions.

File: pkg/Microsoft.NETCore.Platforms/runtimeGroups.props [hunk 3]
```diff
@@ -84,15 +84,20 @@
     <RuntimeGroup Include="ol">
       <Parent>rhel</Parent>
       <Architectures>x64</Architectures>
-      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6</Versions>
+      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6;7.7;7.8;7.9;7.10</Versions>
```
Comment (omajid) [inline pkg/Microsoft.NETCore.Platforms/runtimeGroups.props:87]: Have you seen any errors that require adding these specific `ol` versions? Are there errors building if the parent `rhel` version of an `ol` version is missing?

File: pkg/Microsoft.NETCore.Platforms/runtimeGroups.props [hunk 4]
```diff
@@ -84,15 +84,20 @@
...
-      <Architectures>x64</Architectures>
-      <Versions>8;8.0</Versions>
+      <Architectures>x64;arm64</Architectures>
+      <Versions>8;8.0;8.1;8.2;8.3;8.4;8.5;8.6</Versions>
       <ApplyVersionsToParent>true</ApplyVersionsToParent>
     </RuntimeGroup>
+    <RuntimeGroup Include="ol">
+      <Parent>rhel</Parent>
+      <Architectures>x64;arm64</Architectures>
+      <Versions>9;9.0;9.1</Versions>
```
Comment (omajid) [inline pkg/Microsoft.NETCore.Platforms/runtimeGroups.props:99]: Are you actually planning to build .NET Core 3.1 on Oracle Linux 9? If so, do you have a plan to deal with OpenSSL 3.0 being the only version available to build against?

**Comment by carlossanlop** [comment]
@Mr-Tao I'm closing this PR since the feedback has not been addressed. Feel free to reopen if you can address it.

**Pull request closed without merging**